### PR TITLE
add helm chart gen installation

### PR DESCRIPTION
**Pull request by mauipipe**

### CODE REVIEW

**Comment by harshal-shah** [comment]
+1

File: ci/pipeline.yml [hunk 1]
```diff
@@ -104,6 +104,7 @@ jobs:
           tag_as_latest: true
           tag: master-release-version/version
           dockerfile: source-code/Dockerfile
+          GITHUB_TOKEN: ((github_token))
```
Comment (harshal-shah) [inline ci/pipeline.yml]: This may not work, use `build_args` instead

https://github.com/concourse/docker-image-resource#parameters-1

Comment (mauipipe) [reply ci/pipeline.yml]: it should be fixed, thanks to spot it :)

**Comment by harshal-shah** [comment]
+1

**Comment by fftorres** [comment]
+1

**Pull request merged**